### PR TITLE
feat: enable traefik access logs

### PR DESCRIPTION
**Pull request by mhrabovcin**
**What type of PR is this?**
Feature

**What this PR does/ why we need it**:
Enables by default traefik access logging.

**Which issue(s) this PR fixes**:
no issue

**Special notes for your reviewer**:
The additional load that this logging would put on ELK stack was tested during the last MWT. Logging did not generate additional significant load on the logging stack.

**Does this PR introduce a user-facing change?**:
```release-note
traefik: access log is enabled by default
```

**Checklist**

* [x] The commit message explains the changes and why are needed.
* [x] The code builds and passes lint/style checks locally.
* [ ] The relevant subset of integration tests pass locally.
* [ ] The core changes are covered by tests.
* [ ] The documentation is updated where needed.

### CODE REVIEW

File: addons/traefik/1.7.x/traefik-14.yaml [hunk 1]
```diff
@@ -61,6 +61,8 @@ spec:
           cpu: 1000m
         requests:
           cpu: 500m
+      accessLogs:
+        enabled: false
```
Comment (makkes) [inline addons/traefik/1.7.x/traefik-14.yaml:65]: doesn't this **disable** access logs?

Comment (mhrabovcin) [reply addons/traefik/1.7.x/traefik-14.yaml]: Oh you are right, I've fixed it and changed the value to `true`.

File: addons/traefik/1.7.x/traefik-14.yaml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/name: cert-manager
+  chartReference:
+    chart: traefik
+    repo: https://mesosphere.github.io/charts/staging
+    version: 1.72.20
```
Comment (hectorj2f) [inline addons/traefik/1.7.x/traefik-14.yaml:24]: could we use `1.72.21` which is the latest version ?

Comment (makkes) [reply addons/traefik/1.7.x/traefik-14.yaml:24]: #304 is bumping traefik, anyway so bumping here doesn't seem necessary to me.

**Comment by joejulian** [comment]
Sorry, I can't merge because the commits aren't signed. Please sign them and we should be good to merge.

**Pull request merged**